README.md

```diff
@@ -59,10 +62,7 @@ expect(sum(1, 2)).not.toBeNull();
 This library provides **fluent assertions**, which means you can chain multiple matcher functions to the same value under test:
 
 ```ts
-expect("assertive-ts is awesome!")
-  .toStartWith("assertive-ts")
-  .not.toContain("unsafe")
-  .toEndWith("awesome!");
+expect("assertive-ts is awesome!").toStartWith("assertive-ts").not.toContain("unsafe").toEndWith("awesome!");
```

Comment:
Let's revert this change (maybe caused by an IDE formatter?). The code was intentionally stacked for better readability.
```suggestion
expect("assertive-ts is awesome!")
  .toStartWith("assertive-ts")
  .not.toContain("unsafe")
  .toEndWith("awesome!");
```